src/ucp/api/ucp.h

```diff
@@ -409,6 +428,30 @@ typedef enum {
 } ucp_stream_recv_flags_t;
 
 
+/**
+ * @ingroup UCP_COMM
+ * @brief flags to indicate types of events on a stream associated endpoint.
+ *
+ * This enumeration defines types of events on a stream associated endpoint.
+ * These flags will be set in the field @ref ucp_stream_poll_ep_t::flags of out
+ * parameter @a poll_eps returned by @ref ucp_stream_worker_poll routine.
+ */
+typedef enum {
+    /**
+     * There is data to receive.
+     */
+    UCP_STREAM_POLL_FLAG_IN   = UCS_BIT(0),
```

Comment:
this breaks compatibility

Reply:
will document in commit message and PR desc